serverless.yml

```diff
@@ -0,0 +1,80 @@
+# Welcome to serverless. Read the docs
+# https://serverless.com/framework/docs/
+
+# Serverless.yml is the configuration the CLI
+# uses to deploy your code to your provider of choice
+
+# The `service` block is the name of the service
+service: hathor-explorer-service
+
+# The `provider` block defines where your service will be deployed
+provider:
+  name: aws
+  runtime: python3.8
+  profile: serverless
+  region: ${env:AWS_DEFAULT_REGION}
+  vpc:
+    securityGroupIds:
+      - ${env:AWS_VPC_ID}
+    subnetIds:
+      - ${env:AWS_SUBNET_ID_1}
+      - ${env:AWS_SUBNET_ID_2}
+      - ${env:AWS_SUBNET_ID_3}
+  environment:
+    HATHOR_CORE_DOMAIN: ${env:HATHOR_CORE_DOMAIN}
+    HATHOR_NODES: ${env:HATHOR_NODES}
+    REDIS_KEY_PREFIX: ${env:REDIS_KEY_PREFIX}
+    REDIS_HOST: ${env:REDIS_HOST}
+    REDIS_PORT: ${env:REDIS_PORT}
+    REDIS_DB: ${env:REDIS_DB}
+
+plugins:
+  - serverless-dotenv-plugin
```

Comment:
`Changes in serverless>=3.0.0 means this plugin can no longer preload environment variables into Serverless.`

Maybe we shouldn't be using this plugin since it stops working in serverless v3.
https://www.serverless.com/plugins/serverless-dotenv-plugin

Reply:
Changed to serverless native dotenv: https://github.com/HathorNetwork/hathor-explorer-service/pull/3/files#diff-8cf16232f3b7094ea431234555d38ad71e6e62764c3e29a5841e25172aea5049R9